### ISSUE TITLE
Unintentional Exclusion - Web accessibility and how we’re failing our users

### ISSUE THREAD

**Issue by leekinney** [SassConf/2015-austin-speaker-cfp#78]
# Unintentional Exclusion - Web accessibility and how we’re failing our users
## Type of Presentation (required)

[ x ] Standard Length Talk
or
[ x ] Lightning Talk 
[  ] Workshop
[  ] Moderated Discussion
## Description (required)

Web accessibility is usually thought of as something that is hard to do or expensive to work into a project. It is also thought of as something that is only beneficial to users with severe or noticeable disabilities. Many times accessible designs are a result of good UX practices and can benefit all users. Creating accessible websites can help our aging populations navigate the web easier as well as someone with low literacy or a learning disability. 

If you add up all the users in the world who have a disability, whether mild or severe, long-term or temporary, this number far surpasses the amount of people using IE8. Why do we put more consideration into browser support than we do people with disabilities? 

The web has become an essential source of information and it is our responsibility as designers and developers to take all users into consideration and build a better web experience for everyone. Some key points this talk will cover will be:
- A brief overview of accessible front-end design
- Some simple guidelines to make accessible websites
- Proof that when accessibility is built in at the beginning of a project, it is much less painful and less expensive to implement
- Some simple steps for auditing a website for accessibility 
- Tools, extensions and most importantly, Sass techniques  that will help you to easily incorporate accessibility into your workflow 

You will walk away with a better understanding of how you may be excluding certain users from your sites and immediate actions to prevent it from happening. 
## Speaker Info (required)
- Name      : LeeAnn Kinney
- Location  : Philadelphia, PA
- Contact   : hi@leekinney.is
##### Social Media:
- Twitter   : [@_leekinney](https://twitter.com/_leekinney)
- GitHub    : [leekinney](https://github.com/leekinney)
- Url(s)    : [leekinney.is](http://leekinney.is/)
##### Bio:

LeeAnn is a front-end developer living in Philadelphia. She is a web accessibility advocate, events coordinator for [Girl Develop It Philly](https://www.girldevelopit.com/) and co-organizer of [LadyHacks](http://ladyhacks.org/) and [ELA conf](http://elaconf.com/). In her free time, LeeAnn loves to hike, bike and camp.
##### Photo:

![Avatar](http://leeannkinney.com/img/leeann.jpg)

**Comment by codingdesigner** [comment]
Lee and I did a mentoring session. I think this is going to be a really great session.
